packages/synapse-react-client/src/components/file/upload/BasicFileHandleUpload.tsx

```diff
@@ -8,11 +8,22 @@ import { noop } from 'lodash-es'
 import { ForwardedRef, forwardRef, useEffect, useImperativeHandle } from 'react'
 
 export type BasicFileHandleUploadProps = {
-  allowMultipleUpload: boolean
+  /**
+   * Whether to allow uploading multiple files.
+   * Currently, only single file upload is supported by this UI component.
+   */
+  allowMultipleUpload: false
+  /**
+   * Whether to disable "drag-and-drop to upload" functionality.
+   * Currently, drag-and-drop cannot be enabled.
+   */
+  disableDragAndDrop: true
```

Comment:
Don't allow drag-and-drop for this basic uploader (we can change this to `boolean` if/when we move the logic to SRC!)